packages/perseus/src/widgets/radio/widget.jsx

```diff
@@ -443,28 +475,35 @@ class Radio extends React.Component<Props> {
             // Support legacy choiceState implementation
         }
         /* istanbul ignore if - props.values is deprecated */
-        // $FlowFixMe[prop-missing]
         if (props.values) {
             let noneOfTheAboveIndex = null;
             let noneOfTheAboveSelected = false;
 
-            const values = props.values.slice();
+            const choicesSelected = props.values.map((value) => !!value);
             const countChoices = props.countChoices;
             const numCorrect = props.numCorrect;
+            const valuesLength = props.values?.length || 0;
```

Comment:
**style nit (non-blocking):** It's nice to pair optional chaining and nullish coalescing.  That would make this `props.values?.length ?? 0`.

Reply:
Ooo, also very cool! So if you do an or check, but you're really checking if the left side is null or undefined, then you use nullish coalescing. Fancy! (Updated~)